Deprecate Identity Providers API (#5026)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/idps/index.md

# Identity Providers API

> The Authorization Servers API reference is available at the [Okta API reference portal](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/AuthorizationServer/).

Author: ```suggestion
The Identity Providers API reference is available at the [Okta API reference portal](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/AuthorizationServer/).
```

> Explore the [Okta Public API Collections](https://www.postman.com/okta-eng/workspace/okta-public-api-collections/overview) workspace to get started with the Authorization Servers API Postman collection.

Author: ```suggestion
Explore the [Okta Public API Collections](https://www.postman.com/okta-eng/workspace/okta-public-api-collections/overview) workspace to get started with the Identity Providers API Postman collection.
```

```http
HTTP/1.1 200 OK
```
-->
## Identity Provider key store operations

> The key store operations for the Identity Provider API are available at the [Okta API reference portal](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/IdentityProviderKeys/#tag/IdentityProviderKeys).

Reviewer: ```suggestion
The key store operations for the Identity Providers API are available at the [Okta API reference portal](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/IdentityProviderKeys/#tag/IdentityProviderKeys).
```

> The key store operations for the Identity Provider API are available at the [Okta API reference portal](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/IdentityProviderKeys/#tag/IdentityProviderKeys).

Author: ```suggestion
The key store operations for the Identity Providers API are available at the [Okta API reference portal](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/IdentityProviderKeys/#tag/IdentityProviderKeys).
```

  "x5t#S256": "5GOpy9CQVtfvBmu2T8BHvpKE4OGtC3BuS046t7p9pps"
}
```
-->
### Clone signing Key Credential for IdP

> The Clone signing Key Credential for IDP operation is available at the [Okta API reference portal](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/IdentityProviderSigningKeys/#tag/IdentityProviderSigningKeys/operation/cloneIdentityProviderKey).

Author: ```suggestion
The Clone signing Key Credential for IdP operation is available at the [Okta API reference portal](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/IdentityProviderSigningKeys/#tag/IdentityProviderSigningKeys/operation/cloneIdentityProviderKey).
```

  }
}
```
-->
## Identity Provider User operations

> The Identity Provider User operations are available at the [Okta API reference portal](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/IdentityProviderUsers/).

Author: ```suggestion
The Identity Provider User operations for the Identity Provider API are available at the [Okta API reference portal](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/IdentityProviderUsers/).
```

> The Identity Provider User operations for the Identity Provider API are available at the [Okta API reference portal](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/IdentityProviderUsers/).

Reviewer: ```suggestion
The Identity Provider User operations for the Identity Providers API are available at the [Okta API reference portal](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/IdentityProviderUsers/).
```
